src/common/country-base.ts

```diff
@@ -319,9 +319,10 @@ export const african_countries = [
     'zm',
     'zw',
 ]
-export const isEuCountry = (clients_country) => eu_countries.includes(clients_country)
+export const isEuCountry = (clients_country: string): boolean =>
+    eu_countries.includes(clients_country)
```

Comment:
```suggestion

export const eu_countries = [
    'it',
    'de',
    'fr',
    'lu',
    'gr',
    'mf',
    'es',
    'sk',
    'lt',
    'nl',
    'at',
    'bg',
    'si',
    'cy',
    'be',
    'ro',
    'hr',
    'pt',
    'pl',
    'lv',
    'ee',
    'cz',
    'fi',
    'hu',
    'dk',
    'se',
    'ie',
    'im',
    'gb',
    'mt',
] as const;
type TEuCountries = typeof eu_countries[number];

export const isEuCountry = (clients_country: TEuCountries): boolean =>
    eu_countries.includes(clients_country)

```
how about this approach? 
we get more type safty and cool features of typescript. 
let me know what you think, thanks

Reply:
I exactly prefer your decision. Thanks for share, very smart approach 👍